### ISSUE TITLE
User Description?

### ISSUE THREAD

**Issue by McCouman** [eazyliving/firtz-extensions#1]
Hi habe ich habe versucht ne description zusätzlich einzubauen: => http://firtz-designs.wikibyte.org/firtz-design/show

/**
Internal Server Error
Undefined index: desc

• tmp/2zpzgitqpxkww.2kjwlhff1buoc.php:14 Base->{closure}()
• src/lib/base.php:2325 require()
• src/lib/base.php:2475 View->sandbox()
• tmp/2zpzgitqpxkww.1hv2k3jilpvk0.php:314 Preview->render()
• src/lib/base.php:2325 require()
• src/lib/base.php:2475 View->sandbox()
• src/classes/feed.php:662 Preview->render()
• index.php:358 feed->renderHTML()
• src/lib/base.php:1422 Base->call()
• index.php:696 Base->run()
**/

Wo muss ich denn der Erweiterung den "desc" oder decription erklären das er das ausgeben darf?
Da ich das gerade mal in Angriff nehmen will mit dem designen

**Comment by McCouman** [comment]
Ah, man muss leider "immer" alle Daten zum Benutzer auswählen. Sonst kommt nen Fehler

**Comment by eazyliving** [comment]
Jo, blöde von mir. Idiotisch gerade zu. Ist jetzt alles initialisiert.

**Comment by McCouman** [comment]
OK am Wochenende werde ich das CSS noch einmal auseinander nehmen. Da sind viele Fehler drin. Da mache ich gleich das CSS für die Contributors schick^^. Musste leider noch mal installieren. Neue Instanz und so. Hatte vergessen ne Sicherung zu machen /*Kopf vor Wand*/

Ist ne absolut geile Sache mit den Extensions!
